rpcserver.go

```diff
@@ -1944,6 +1944,16 @@ func handleGetBlockchainInfo(_ context.Context, s *rpcServer, cmd interface{}) (
 			aInfo := types.AgendaInfo{
 				StartTime:  agenda.StartTime,
 				ExpireTime: agenda.ExpireTime,
+				Status: blockchain.ThresholdStateTuple{
```

Comment:
Not that performance is a huge concern here, but it seems like it would be more efficient (and probably read slightly better too) to create this outside of the loop in a local and just set it here vs building a struct each time.

e.g.

```Go
defaultStatus := blockchain.ThresholdStateTuple{
	State: blockchain.ThresholdDefined,
}.String()
for version, deployments := := range params.Deployments {
...
			Status: defaultStatus,
...
}
```

Reply:
Agreed, updated.